src/plots/cartesian/axes.js

```diff
@@ -1792,7 +1795,9 @@ axes.doTicks = function(gd, axid, skipTitle) {
         }
         else {
             flipit = (axside === 'right') ? 1 : -1;
-            labely = function(d) { return d.dy + d.fontSize / 2 - labelShift * flipit; };
+            labely = function(d) {
+                return d.dy + d.fontSize * 0.35 - labelShift * flipit;
```

Comment:
If you run into such a thing I'm interested as the upcoming `table` uses it. IIRC `dominant-baseline` had some inconsistencies but don't now recall an issue with `alignment-baseline`. We'll see, I just bought a machine for real-life testing which as MS Edge on it.